packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -591,7 +601,14 @@ export async function downloadDocsAndUpsertFHIR({
           return FHIRDocRef;
         } catch (error) {
           errorCount++;
-          if (isConvertibleDoc) errorCountConvertible++;
+
+          if (file) {
+            const isFileConvertible = fileIsConvertible(file);
+            if (isFileConvertible) errorCountConvertible++;
+          } else {
+            const isConvertibleDoc = isConvertible(doc.content.mimeType);
+            if (isConvertibleDoc) errorCountConvertible++;
+          }
 
```

Comment:
The logic above includes both file and document because technically the upload can fail so then i want to fallback to the document to see if it should have been converted.